.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
this file should not be included...  we can add it to .gitignore if we need too...

Reply:
let me know if we need a formal request for this .gitignore, please add if informal is ok